### ISSUE TITLE
Terminal encoding and `dvc run`

### ISSUE THREAD

**Issue by efiop** [dataversioncontrol/dvc#686]
@dmpetrov discovered an issue with `dvc run` when it throws an error(same w/o -v):
```
Unexpected error: 'ascii' codec can't decode byte 0xef in position 137272: ordinal not in range(128)
```

Adding 
```
import sys
reload(sys)
sys.setdefaultencoding('utf-8')
```
helps.

Probably just python2 problem, as python3 uses utf8 by default instead of ascii.

**Comment by itcarroll** [comment]
+1

I just encountered this doing the tutorial at the first `dvc run` step:
```
$ dvc run -d data/Posts.xml.tgz -o data/Posts.xml tar -zxf data/Posts.xml.tgz -C data/
Using 'Posts.xml.dvc' as a stage file
Reproducing 'Posts.xml.dvc':
	tar -zxf data/Posts.xml.tgz -C data/
Unexpected error: 'ascii' codec can't decode byte 0xef in position 137272: ordinal not in range(128)
```

Is there a hot fix? Like incorporating that setdefaultencoding('utf-8') somewhere?

My installation:
```
$ brew info dvc
dataversioncontrol/dvc/dvc: stable 0.9.6
Git for data science projects
https://dataversioncontrol.com/
/usr/local/Cellar/dvc/0.9.6 (9,515 files, 118.9MB) *
  Built from source on 2018-05-15 at 16:56:22
From: https://github.com/dataversioncontrol/homebrew-dvc/blob/master/Formula/dvc.rb
==> Dependencies
Build: pkg-config ✔
Required: python ✔
[icarroll@sesyncmaclap06] classify $ 
```

**Comment by efiop** [comment]
Hi @itcarroll !

Thank you for your feeback! I am working on it right now. Should be ready soon and released in 0.9.7(released until tomorrow).
